NAMES.md

```diff
@@ -18,4 +18,6 @@
 
 - [Olayiwola Olanrewaju](https://github.com/larrycoal)
 
+- [Chinyere Nwalie] (https://github.com/Chinyere-nwalie)♾💫
+
```

Comment:
Did you see a preview of the Readme after you added your name?

It looks a bit broken, can you remove the space in between the [name] and (link)?

Reply:
Alright. Noted.